bluesky_queueserver/start_manager.py

```diff
@@ -0,0 +1,163 @@
+from multiprocessing import Pipe
+import threading
+import time as ttime
+import pprint
+
+from .worker import RunEngineWorker
+from .manager import RunEngineManager
+
+import logging
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Now I understand why you were seeing overly-verbose log messages from ophyd. This configuration setting will add a handler that displays DEBUG-level messages from _all_ libraries. Library code should not configure handlers like this, at least not at import time. We did this in earlier versions of our libraries and learned to regret it. (Also see the gray "Note" box in the documentation section https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library for some official validation of that opinion.)

This could be added inside the `if __name__ == '__main__':` block (which is more an application than a library) and perhaps be configurable via a verbosity option like `-v`, as we do [in caproto](https://github.com/caproto/caproto/blob/4033c02da7815f7145c4a42581ddf0f76da4bbca/caproto/commandline/get.py#L140-L145).